core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/treemap/TreeMapVisualizerOpDesc.scala

```diff
@@ -0,0 +1,116 @@
+package edu.uci.ics.texera.workflow.operators.visualization.treemap
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{
+  InputPort,
+  OperatorGroupConstants,
+  OperatorInfo,
+  OutputPort
+}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{
+  Attribute,
+  AttributeType,
+  OperatorSchemaInfo,
+  Schema
+}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+//type constraint: value can only be numeric
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "value": {
+      "enum": ["integer", "long", "double"]
+    }
+  }
+}
+""")
+class TreeMapVisualizerOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+
+  @JsonProperty(value = "value", required = true)
+  @JsonSchemaTitle("Value Column")
+  @JsonPropertyDescription("the value associated with each tree node")
+  @AutofillAttributeName
+  var value: String = ""
+
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Hierarchy Path")
+  @JsonPropertyDescription("hierarchy of sectors, from root to leaves")
+  var hierarchy: List[HierarchySection] = List()
```

Comment:
It will be good to include some examples. possibly as a "?" icon to show how to config this operator.

Reply:
will do it later because now we are hardcoding those tutorials on the frontend, which is hard to maintain.